Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/DriveBase.java

```diff
@@ -113,6 +131,28 @@ public void log() {
 		// SmartDashboard.putNumber("Right Distance", this.rightEncoder.get());
 	}
 
+	public double[] distanceTraveled() {
+		// TODO: This would be where the encoder logic would go to determine how far it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)